crates/sqllogictest/testdata/schedules/df_test.toml

```diff
@@ -20,4 +20,8 @@ df = { type = "datafusion" }
 
 [[steps]]
 engine = "df"
-slt = "df_test/show_tables.slt"
\ No newline at end of file
+slt = "df_test/show_tables.slt"
+
+[[steps]]
```

Comment:
nit: One schedule is used for a complete test case, maybe it's better to put it in another schedule?

Reply:
In fact, it's not designed for it. But for simple case it's fine for now.